mailpoet/lib/Config/Menu.php

```diff
@@ -449,19 +449,17 @@ private function registerMailPoetSubMenuEntries() {
     );
 
     // Upgrade page
-    if ($this->capabilitiesManager->showUpgradePage()) {
-      $this->wp->addSubmenuPage(
-        self::MAIN_PAGE_SLUG,
-        $this->setPageTitle(__('Upgrade', 'mailpoet')),
-        esc_html__('Upgrade', 'mailpoet'),
-        AccessControl::PERMISSION_ACCESS_PLUGIN_ADMIN,
-        self::UPGRADE_PAGE_SLUG,
-        [
-          $this,
-          'upgrade',
-        ]
-      );
-    }
+    $this->wp->addSubmenuPage(
+      $this->capabilitiesManager->showUpgradePage() ? self::MAIN_PAGE_SLUG : self::HELP_PAGE_SLUG,
```

Comment:
Since we are not using tiers anymore, do we need to update this method https://github.com/mailpoet/mailpoet/blob/a8f4de675d1482a76ab4505665e64929670f2120/mailpoet/lib/Util/License/Features/CapabilitiesManager.php#L141-L142?

Reply:
This is still needed, because existing customers on Basic/Standard plans don't have full access to all premium features, so they still need to see the Upgrade page so they can upgrade to Business to get access to everything. But I removed related, no longer used method in https://github.com/mailpoet/mailpoet/pull/6199/commits/d6ffd3ffbb8e6fc50f63ffe20b37e29a2903eeaf.